services/slack/slack.py

```diff
@@ -0,0 +1,63 @@
+import json
+import traceback
+
+import tornado.ioloop
+import tornado.web
+import tornado.escape
+from tornado.httpclient import AsyncHTTPClient
+
+from baselayer.app.env import load_env
+from baselayer.log import make_log
+
+env, cfg = load_env()
+log = make_log('slack')
+
+
+class MainHandler(tornado.web.RequestHandler):
+    def set_default_headers(self):
+        self.set_header("Content-Type", 'application/json')
+
+    def error(self, code, message):
+        self.set_status(code)
+        self.write({'message': message})
+
+    def get(self):
+        self.write({'status': 'active'})
+
+    async def post(self):
```

Comment:
In this version, POSTs are not using a `tornado.queue.` This can be a TODO if needed.